corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -459,5 +460,67 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
         });
     };
 
+    Utils.setSyncInterval = function (appId, restartInterval) {
+        hqRequire(["cloudcare/js/formplayer/app"], function (FormplayerFrontend) {
+            const currentApp = FormplayerFrontend.getChannel().request("appselect:getApp", appId);
```

Comment:
Where does this request go? I don't see a reply function for it, but maybe I'm missing something. Locally I swapped in `currentApp = AppsAPI.getAppEntity(appId)` and that seemed to work, but otherwise `currentApp` kept coming back as `undefined`

Reply:
Looks like `appselect:getApp` was removed in this commit `3b5f59be26a17d3052163ddf9c42c546fe1b4db1` and you're right, that this line should have been updated to `currentApp = AppsAPI.getAppEntity(appId)`

Reply:
That explains it. I can make a PR for that. I'm investigating some QA bugs for case search, so that's why I'm looking through old PR's.